src/api/cloud/project/dataprocessing/jobs/cloud-project-dataprocessing-jobs.iceberg.service.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Interceptor purges serviceName from POST payload
+ * @type {{request(*): *}}
+ */
+const postInterceptor = {
+  request(req) {
+    delete req.data.serviceName;
+    return req;
+  },
+};
+
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectDataProcessingJobsIceberg', (iceberg) => iceberg('/cloud/project/:serviceName/dataProcessing/jobs/:jobId', {
+    serviceName: '@serviceName',
+    jobId: '@jobId',
+  }, {
+    post: {
+      method: 'POST',
+      url: '/cloud/project/:serviceName/dataProcessing/jobs',
+      interceptor: postInterceptor,
+    },
+    delete: {
+      method: 'DELETE',
+      url: '/cloud/project/:serviceName/dataProcessing/jobs/:jobId',
+      transformResponse: [],
```

Comment:
Is `transformResponse` still needed?

Reply:
Yes, we return an HTTP 204 (no content) but angular expects a json payload (as stated by default transformResponse). So here we specifically tell angular not to run any transformer on the response.

Reply:
Thanks for the explanation.